google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -960,6 +963,15 @@ def _convert_service_config(self, service_config):
 
         self.session.add(resource)
 
+    def _convert_bigquery_table(self, table):
```

Comment:
It seems like we could remove a lot of these "convert" functions that just directly call `_convert_resource`. Those that only call with `cached=False` can simply use `_convert_resource` directly in the `handlers` dictionary. For those with `cached=True`, we could define a method like:
```
  def _convert_resource_cached(self, resource):
    self._convert_resource(resource, cached=True)
````

This can be used by bucket, dataset, gae_resource, kubernetes cluster, etc.

It also means you can get rid of the `pylint: disable=too-many-lines`

Reply:
Should I make this modification in another pull request?

Reply:
Up to you